src/app/Http/Controllers/VolunteerController.php

```diff
@@ -204,7 +205,8 @@ public function store(Request $request) {
         $data = $request->all();
         $rules = [
             'organisation_id' => 'required',
-            'email' => 'required|string|email|max:255|unique:volunteers.volunteers',
+//            'email' => 'required|string|email|max:255|unique:volunteers.volunteers',
```

Comment:
Please remove old line and a comment to explain what the login of "VolunteerEmail" rule.

Reply:
ok